lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -164,7 +164,11 @@ def add_copy_resources_script_phase
             phase = create_or_update_build_phase(native_target, phase_name)
             script_path = target.copy_resources_script_relative_path
             phase.shell_script = %("#{script_path}"\n)
-            # TODO: Add input and output paths for the copy resources script phase.
+            resource_paths_by_config = target.resource_paths_by_config
+            unless resource_paths_by_config.empty?
+              phase.input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
+              phase.output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
```

Comment:
I chose the lazy way because different resources produce different types of artifacts when compiled. Also I do not believe resources change that often when someone is compiling their code and making changes to sources so it should be fine.